src/main/resources/application.properties

```diff
@@ -2,20 +2,27 @@ server.port=8089
 spring.datasource.url=jdbc:postgresql://${PGHOST}:${PGPORT}/${PG_DATABASE}
 spring.datasource.username=${PG_RW_USER}
 spring.datasource.password=${PG_RW_PASSWORD}
+
 spring.datasource.platform=elrr
 spring.jpa.properties.hibernate.default_schema=elrr
 spring.jpa.show-sql=true
 
 logging.level.org.springframework.security=INFO
+logging.level.org.springframework=INFO
+logging.level.com.deloitte=INFO
+logging.level.root=INFO
 
 cronExpression=0 0/1 * * * *
 
 brokerUrl=elrr-kafka:9092
+
 kafka.topic=test-1
 kafka.groupId=test-consumer-group
 
-server.use-forward-headers=true
+##server.use-forward-headers=true
```

Comment:
this is the production properties, so we want those settings in prod. They get commented out in local.